### PR TITLE
cmds/core/tty: tty command

### PR DESCRIPTION
**Pull request by binjip978**

### CODE REVIEW

File: cmds/core/tty/tty.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"syscall"
+)
+
+func run(stdin io.Writer) error {
```
Comment (rminnich) [inline cmds/core/tty/tty.go:12]: stdin io.Writer?

File: cmds/core/tty/tty.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	fi, err := os.Stdin.Stat()
```
Comment (rminnich) [inline cmds/core/tty/tty.go:13]: did you want to use stdin here, or os.Stdin?

File: cmds/core/tty/tty.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+	if err != nil {
+		return err
+	}
+
+	s, ok := fi.Sys().(*syscall.Stat_t)
```
Comment (rminnich) [inline cmds/core/tty/tty.go:18]: This is a tad confusing. You can stat stdin, but the extended info is not available. Also, can you wrap os.ErrNotExist here? Something like
fmt.Errorf("stat stdin:Sys() is empty:%w", os.ErrNotExist)
for testing?

File: cmds/core/tty/tty.go (hunk 1 above)
Comment (rminnich) [reply cmds/core/tty/tty.go:12]: Maybe it should be
stdin os.File
?

Comment (binjip978) [reply cmds/core/tty/tty.go:16]: oh I meant os.Stdout :)

File: cmds/core/tty/tty.go (hunk 2 above)
Comment (binjip978) [reply cmds/core/tty/tty.go:20]: I want os.Stdin, stdin renamed to stdout.

File: cmds/core/tty/tty.go (hunk 3 above)
Comment (binjip978) [reply cmds/core/tty/tty.go:25]: done, but can't come up with ideas how to use it in tests. Any other tests ideas are welcome!

**Review by rminnich — APPROVED** [review]

**Review by rminnich — APPROVED** [review]

**Pull request merged**